sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -22,107 +17,151 @@
+        self._client = AzureAttestationRestClient(credential, endpoint, **kwargs)
         self._statelock = Lock()
         self._signing_certificates = None
 
     @distributed_trace
-    def get_openidmetadata(self, **kwargs):
-        #type:(Dict[str, Any]) -> Any
-        """ Retrieves the OpenID metadata configuration document for this attestation instance.
+    def get_open_id_metadata(self, **kwargs):
+        # type: (Dict[str, Any]) -> Dict[str, Any]
+        """Retrieves the OpenID metadata configuration document for this attestation instance.
+
+        The metadata configuration document is defined in the `OpenID Connect
+        Discovery <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationResponse>`
+        specification.
+
+        The attestation service currently returns the following fields:
+        * issuer
+        * jwks_uri
+        * claims_supported
 
         :return: OpenID metadata configuration
-        :rtype: Any
+        :rtype: Dict[str, Any]
         """
         return self._client.metadata_configuration.get(**kwargs)
 
     @distributed_trace
-    def get_signing_certificates(self, **kwargs): 
-        # type: (Any) -> list[AttestationSigner]
-        """ Returns the set of signing certificates used to sign attestation tokens.
+    def get_signing_certificates(self, **kwargs):
+        # type: (Any) -> List[AttestationSigner]
+        """Returns the set of signing certificates used to sign attestation tokens.
 
         :return: A list of :class:`azure.security.attestation.AttestationSigner` objects.
 
-        :rtype: list[azure.security.attestation.AttestationSigner]
+        :rtype: List[~azure.security.attestation.AttestationSigner]
 
-        For additional request configuration options, please see `Python Request Options <https://aka.ms/azsdk/python/options>`_.
+        For additional request configuration options, please see `Python Request
+        Options <https://aka.ms/azsdk/python/options>`_.
 
         """
         signing_certificates = self._client.signing_certificates.get(**kwargs)
         signers = []
         for key in signing_certificates.keys:
             # Convert the returned certificate chain into an array of X.509 Certificates.
-            signers.append(AttestationSigner._from_generated(key))
+            signers.append(
+                AttestationSigner._from_generated(  # pylint: disable = protected-access
+                    key
+                )
+            )
         return signers
 
     @distributed_trace
-    def attest_sgx_enclave(self, quote, inittime_data=None, runtime_data=None, **kwargs):
-        # type:(bytes, AttestationData, AttestationData, Dict[str, Any]) -> AttestationResponse[AttestationResult]
-        """ Attests the validity of an SGX quote.
+    def attest_sgx_enclave(
+        self,
+        quote,  # type: bytes
+        **kwargs  # type: Dict[str, Any]
+    ):  # type: (...) -> Tuple[AttestationResult, AttestationToken]
```

Comment:
It's more typical in my experience to do something like this, but I don't think it's particularly important (just shorter):
```suggestion
    def attest_sgx_enclave(self, quote, **kwargs):
        # type: (bytes, **Any) -> Tuple[AttestationResult, AttestationToken]
```